updated default mysql auth command & some slight md tweaks (#855)
echoCTF/echoCTF.RED: docs/INSTALL-LINUX.md

cp frontend/config/db-local.php frontend/config/db.php
```

**NOTE:** If you keep the default db.php on the applications, update the mysql authentication plugin to allow root access
```sh

> mysql -e "ALTER USER root@localhost IDENTIFIED VIA mysql_native_password; SET PASSWORD = PASSWORD('')" mysql

Reviewer: ```suggestion
mysql -e "ALTER USER root@localhost IDENTIFIED VIA mysql_native_password" mysql
mysql -e "SET PASSWORD = PASSWORD('')" mysql
```

```

> If that didn't work, you must likely have MariaDB < 10.4.3, try this one instead

Reviewer: ```suggestion
For older versions of MariaDB < 10.4.3, you can try the following instead
```
